packages/components/src/InsightsLabel/InsightsLabel.tsx

```diff
@@ -6,19 +6,19 @@ import { CriticalRiskIcon } from '@patternfly/react-icons';
 import { EqualsIcon } from '@patternfly/react-icons';
 import { Label, LabelProps } from '@patternfly/react-core';
 import React from 'react';
-import classNames from 'classnames';
 
 type InsightsLabelValueMapping = {
   [key: number]: {
     icon: React.ReactNode;
     text: string;
+    class: string;
   };
 };
 const VALUE_TO_STATE: InsightsLabelValueMapping = {
-  1: { icon: <AngleDoubleDownIcon />, text: 'Low' },
-  2: { icon: <EqualsIcon />, text: 'Moderate' },
-  3: { icon: <AngleDoubleUpIcon />, text: 'Important' },
-  4: { icon: <CriticalRiskIcon />, text: 'Critical' },
+  1: { icon: <AngleDoubleDownIcon />, text: 'Low', class: 'pf-c-label pf-m-blue' },
+  2: { icon: <EqualsIcon />, text: 'Moderate', class: 'ins-c-label-2' },
+  3: { icon: <AngleDoubleUpIcon />, text: 'Important', class: 'pf-c-label pf-m-orange' },
+  4: { icon: <CriticalRiskIcon />, text: 'Critical', class: 'pf-c-label pf-m-red' },
```

Comment:
Do now use class. Use the LabelComponent props API instead. For the colors it should be the `color` prop. https://www.patternfly.org/v4/components/label-group/#label.

Reply:
Ok, sure